docs/notebooks/adding_sources.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
Can you clear the outputs for this notebook and the ones below? We do not include output in the github version of the notebooks to keep the space small.